code/src/components/Playlist.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
The playlist is such a nice feature and I like that you styled it completely different from the rest of the page. Would love to hear more about how you've done this